docs/docs/07-cli.md

```diff
@@ -18,9 +18,7 @@ test, build, and launch your blockchain.
 
 To get started, create a blockchain:
 
-```
-ignite scaffold chain mars
-```
+ignite scaffold chain github.com/username/mars
```

Comment:
Oups, maybe a problem with a merge ?

Reply:
The `07-cli.md` is generated from the CLI documentation and this specific section comes from [here](https://github.com/ignite/cli/blob/develop/ignite/cmd/cmd.go#L54). What I am not sure is if it's a good idea to change this to single name in this case. I personally prefer the GitHub URI here.

Reply:
ok for me